WooCommerce/Classes/ViewRelated/Products/AI/ProductDescriptionGenerationViewModel.swift

```diff
@@ -80,6 +80,10 @@ final class ProductDescriptionGenerationViewModel: ObservableObject {
     func handleFeedback(_ vote: FeedbackView.Vote) {
         analytics.track(event: .AIFeedback.feedbackSent(source: .productDescription,
                                                         isUseful: vote == .up))
+        if vote == .down {
+            languageIdentifiedUsingAI = nil
+        }
```

Comment:
Nit: Please consider adding a comment here to explain why the language is reset. It's worth it to mention that this is an experiment too.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10177/commits/d48b678b2433243e93080abd6aadee705a5c8f79